packages/cli/test/utils/crucible/kurtosis/runner/loadKurtosisConfig.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Loader for YAML input into a KurtosisNetworkConfig
+ * Used with *.test.ts files and during the testing phase
+ */
+
+// Loader for YAML input into a KurtosisNetworkConfig
+// Used with *.test.ts files and during the testing phase
+
+import fs from "node:fs/promises";
+import path from "node:path";
+import {parse} from "yaml";
+import type {KurtosisNetworkConfig} from "../runner/kurtosisTypes.js";
+
+export async function loadKurtosisConfig(fileName: string, baseDir?: string): Promise<KurtosisNetworkConfig> {
+  let fullPath: string;
+
+  if (baseDir) {
+    // If baseDir is provided, treat fileName as relative to baseDir
+    fullPath = path.join(baseDir, fileName);
+  } else {
+    // If no baseDir, treat fileName as absolute path
+    fullPath = fileName;
+  }
+
+  const raw = await fs.readFile(fullPath, "utf8");
+  return parse(raw) as KurtosisNetworkConfig;
```

Comment:
Is there a way to validate the kurtosis config at this point? may be some helper function in their SDK.